reviews/sandbox/2023/2023-submariner-annual.md

```diff
@@ -0,0 +1,100 @@
+# Submariner 2023 CNCF Sandbox Annual Review
+
+## Devstats
+
+Submariner's rate of contributions is stable year-over-year and increasing over the life of the project.
+
+Year:
+
+https://submariner.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=1d&from=now-1y&to=now-1h
+
+Project lifetime:
+
+https://submariner.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=1d&from=now-5y&to=now-1h
+
+## Maintainers
+
+Submariner has 4 [Owners](https://submariner.io/community/contributor-roles/#project-owner) from 2 companies,
+5 [Committers](https://submariner.io/community/contributor-roles/#committers) from 1 company,
+and 10 [Members](https://submariner.io/community/contributor-roles/#member) from 3 companies.
+
+Owners:
+
+* Chris Kim (@Oats87, SUSE)
+* Sridhar Gaddam (@sridhargaddam, Red Hat)
+* Stephen Kitt (@skitt, Red Hat)
+* Tom Pantelis (@tpantelis, Red Hat)
+
+https://github.com/orgs/submariner-io/teams/submariner-owners
+
+Committers:
+
+* Aswin Suryanarayanan (@aswinsuryan, Red Hat)
+* Daniel Farrell (@dfarrell07, Red Hat)
+* Janki Chhatbar (@Jaanki, Red Hat)
+* Mike Kolesnik (@mkolesnik, Red Hat)
+* Vishal Thapar (@vthapar, Red Hat)
+
+https://github.com/submariner-io/submariner/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/submariner-operator/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/admiral/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/cloud-prepare/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/lighthouse/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/subctl/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/shipyard/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/releases/blob/devel/CODEOWNERS.in
+https://github.com/submariner-io/submariner-website/blob/devel/CODEOWNERS.in
+
+Members:
+
+* Andrew Stoycos (@astoycos, Red Hat)
+* Brent Salisbury (@nerdalert, Red Hat)
+* Maayan Friedman (@maayanf24, Red Hat)
+* Nir Yechiel (@nyechiel, Red Hat)
+* Noam Manos (@manosnoam, Red Hat)
+* Masaki Kimura (@mkimuram, Hitachi)
+* Maxim Babushkin (@MaxBab, Red Hat)
+* pkomarov (@pinikomarov, unknown)
```

Comment:
@skitt said this member is a Red Hatter and I confirmed. Will send an edit to fix.